### ISSUE TITLE
Limited requests for Global Tagging

### ISSUE THREAD

**Issue by nrocaalh** [IBM/platform-services-python-sdk#197]
We detected a problem with an application that is not working properly for the limited requests on the global tagging api, we're using the sdk to use this api.

We would like to upgrade the sdk Global tagging version to version v3.
The actual version v1 is limited to 100 requests per user in one minute.
The requested version is limited to 500 requests per user in one minute.

 https://cloud.ibm.com/docs/account?topic=account-tag&interface=api#search-tags-api-link

**Comment by padamstx** [comment]
It turns out that the global tagging service within the platform services SDKs is in fact up to date.   The individual operations that make up the service use path strings that start with "/v3" (indicating v3 of the service), while the overall version number within the API definition is still listed as only 1.2.0 (this should probably reflect 3.x.x at this point):
```
info:
  ...
  title: Global Tagging
  version: 1.2.0
```
So the overall service is given "v1" while the individual operations are in fact associated with "v3".

cc: @damianovesperini

**Comment by nrocaalh** [comment]
Hi Phil,
Thanks for the information. 
We have checked the code and we observe the function list_tags is using the V3 version https://github.com/IBM/platform-services-python-sdk/blob/main/ibm_platform_services/global_tagging_v1.py#L174 . 
We also have checked documentation and it would be more convinient to use the "get tags" api instead of the function "list_tags"? Here I'm attaching the api link documentation https://cloud.ibm.com/docs/account?topic=account-tag&interface=api#search-tags-api-link 
Thanks again.

**Comment by padamstx** [comment]
The list_tags function in the Python SDK is using `GET /v3/tags` which is what is referred to as "get tags" in the document you linked.
The potential confusion is the fact that the SDK code is actually generated from the service's API definition and the function names are obtained directly from the "operationId" field for each operation.   
For example, in the tagging API defn, we have this:
```
paths:
  /v3/tags:
    get:
      x-swagger-router-controller: swagger_controllers
      operationId: list_tags
      tags:
        - Tags
      summary: Get all tags
      description: >
        Lists all tags that are in a billing account. Use the `attached_to` parameter to
        return the list of tags that are attached to the specified resource.

```
which means the generated function for the `GET /v3/tags` operation will be "list_tags" (Python), "ListTags" (Go), or "listTags" (Java, Node).

So, bottom line is that list_tags function is in fact the "get tags" operation.

**Comment by padamstx** [comment]
Actually, no I can't make that sort of change.  That would need to be done by the team responsible for the tagging service.  However, I think the solution to this issue is to simply use the Global Search service's "search" operation instead of the "list_tags" operation from the Global Tagging service.   This is the suggestion found in https://cloud.ibm.com/docs/account?topic=account-tag&interface=api#search-tags-api-link, where it mentions that the Tagging "list_tags" operation has a much lower rate limit than the Search service's "search" operation.
